### PR TITLE
Update stars of redismodule-ratelimit + JSON formatting

### PR DESCRIPTION
**Pull request by linfangrong**
format

### CODE REVIEW

File: modules.json
Comment (zuiderkwast) [inline modules.json:318]: This matches the github stars of the project. Verified.

**Review by zuiderkwast — APPROVED** [review]

**Comment by madolson** [comment]
Would prefer to not merge this unless we plan on implementing a checker to validate the format of the various JSON files.

**Comment by zuiderkwast** [comment]
@madolson Do you mean the whitespace changes or the number of stars? I guess it'd possible to check/automate both...

**Comment by madolson** [comment]
@zuiderkwast  yes

**Comment by zuiderkwast** [comment]
For the stars, issue #833 (from 2017) and it's seems to be fixed by this PR #1030 (from 2018). Maybe it still works and can be merged....

**Comment by zuiderkwast** [comment]
@linfangrong We will try to make the redis.io website update the stars automatically from the GitHub API. I don't think we want to merge only whitespace fixes.

**Comment by itamarhaber** [comment]
As @zuiderkwast said, stars are no longer needed here. Closing.

**Pull request closed without merging**
